servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -7231,8 +7231,10 @@ void logPrincipal(ResourceContext ctx) {
     }
 
     public ResourceContext newResourceContext(HttpServletRequest request,
-                                              HttpServletResponse response) {
-        Object timerMetric = metric.startTiming("zms_api_latency", null);
+                                              HttpServletResponse response,
+                                              String httpMethod,
+                                              String apiName) {
+        Object timerMetric = metric.startTiming("zms_api_latency", null, null, httpMethod, apiName);
```

Comment:
Getting the principal domain will require me to start the timer after the principal is authenticated (so not when creating the context). 
For now, I've decided to leave this out along with the request domain.
This will only affect implementers that get the tags at the start of the timer (we send all arguments when stopping the timer)